sps30-i2c/sps30.h

```diff
@@ -211,15 +219,55 @@ int16_t sps30_start_manual_fan_cleaning();
  *
  * The caller should wait at least SPS30_RESET_DELAY_USEC microseconds before
  * interacting with the sensor again in order for the sensor to restart.
- * Interactions with the sensor without this delay might fail.
+ * Interactions with the sensor before this delay might fail.
  *
  * Note that the interface-select configuration is reinterpreted, thus Pin 4
  * must be pulled to ground during the reset period for the sensor to remain in
  * i2c mode.
  *
  * Return:          0 on success, an error code otherwise
  */
-int16_t sps30_reset();
+int16_t sps30_reset(void);
+
+/**
+ * sps30_sleep() - Send the (idle) sensor to sleep
+ *
+ * The sensor will reduce its power consumption to a minimum, but must be woken
+ * up again with sps30_wake_up() prior to resuming operations. It will only
+ * suceed if the sensor is idle, i.e. not currently measuring.
+ * Note that this command only works on firmware 2.0 or more recent.
```

Comment:
@winkj: @psachs Mentioned that this wording may be confusing here. Can you take a look at it?

Reply:
How about his:

> sps30_sleep() - Put the sensor in sleep (idle) mode
> 
> Puts the sensor into sleep/idle mode to reduce power. This call will fail
> if the sensor is not idle (i.e. currently measuring).
> To leave sleep mode and enter the regular operation again,
> sps30_wake_up() must be called.

Reply:
Alternative (more verbose):

>  sps30_sleep() - Put the sensor in sleep (idle) mode
> 
> 
> Puts the sensor into sleep/idle mode to reduce power. This call 
> will only succeed if the sensor is in idle mode; if the sensor is 
> measuring, make sure to call sps30_stop_measurement() first.
> 
> To leave sleep mode and enter the regular operation again,
> sps30_wake_up() must be called.